.github/workflows/cargo_bloat.yml

```diff
@@ -0,0 +1,83 @@
+name: Binary size report
+on:
+  push:
+    branches: [ develop ]
+    paths:
+      - 'libraries/**/*.rs'
+      - 'src/**/*.rs'
+      - 'patches/**'
+      - '**/Cargo.toml'
+      - '.cargo/config'
+      - '!third_party/**'
+  pull_request:
+    branches: [ develop ]
+    types: [opened, synchronize, reopened]
+
+jobs:
+  cargo_bloat:
+    runs-on: ubuntu-18.04
+    steps:
+      # First run: PR
+      - uses: actions/checkout@v2
+        with:
+          submodules: "true"
+      - name: Install Rust toolchain
+        run: rustup show
+      - uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: Install Python dependencies
+        run: python -m pip install --upgrade pip setuptools wheel
+      - name: Set up OpenSK
+        run: ./setup.sh
+      - uses: actions-rs/cargo@v1
+        with:
+          command: install
+          args: cargo-bloat
+      - name: Run bloat on the PR
+        run: .github/workflows/bloat_output.sh "Output for cargo bloat including the PR" "bloat_output.txt"
+
+      - name: Copy script
+        run: cp .github/workflows/bloat_output.sh ${{ runner.temp }}/bloat_output.sh
+      - name: Copy text
+        run: cp bloat_output.txt ${{ runner.temp }}/bloat_output.txt
+
+      # Second run: PR
+      - uses: actions/checkout@v2
+        with:
+          submodules: "true"
+          ref: "develop"
+          path: OpenSK_develop
+      - name: Install old Rust toolchain
+        run: rustup show
+      - uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: Install Python dependencies
+        run: python -m pip install --upgrade pip setuptools wheel
+      - name: Set up OpenSK
+        working-directory: ./OpenSK_develop
+        run: ./setup.sh
+      - uses: actions-rs/cargo@v1
+        with:
+          command: install
+          args: cargo-bloat
+      - name: Copy script back
+        working-directory: ./OpenSK_develop
+        run: cp ${{ runner.temp }}/bloat_output.sh .github/workflows/bloat_output.sh
+      - name: Copy text back
+        working-directory: ./OpenSK_develop
+        run: cp ${{ runner.temp }}/bloat_output.txt .github/workflows/bloat_output.txt
```

Comment:
I'm not sure I understand why we need to copy files around so much.

Why not always use `${{ runner.temp }}/bloat_output.txt` each time we need this file? This file should never be copied.

Why not use `../.github/workflows/bloat_output.sh` from withing `OpenSK_develop`? This file should never be copied.

Reply:
Fixed with the new script, that takes both outputs and generates the comment from there.